cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/AsyncKafkaCruiseControl.java

```diff
@@ -128,7 +128,7 @@ public OperationFuture getBrokerStats(ClusterLoadParameters parameters) {
   public OperationFuture getProposals(ProposalsParameters parameters) {
     OperationFuture future = new OperationFuture("Get customized proposals");
     pending(future.operationProgress());
-    _sessionExecutor.submit(new GetProposalsRunnable(this, future, parameters));
+    _sessionExecutor.submit(new GetProposalsRunnable(this, future, parameters, _config));
```

Comment:
Is it intentional that we do not pass `_config` to some responses in this class -- e.g. `GetBrokerStatsRunnable`?

Reply:
No, I missed this one. Really good catch!